docs/02_installation/README.md

```diff
@@ -31,7 +31,7 @@ like contributing back to the project.
 ### Python
 
 EasyBuild is implemented in Python, and is **compatible with both Python 2.7 and Python 3.5+**
-(that is, Python 3.5 or a newer version of Python 3).
+(that is, Python 3.5 or a newer version of Python 3). The installed Python must support SSL.
```

Comment:
If we mention that SSL support is required, we should also mention how to check for it. But won't most Python installations have this anyway?

Reply:
I think you'd have to try hard to avoid SSL support these days, even when building it yourself. If you're that smart, you can be that smart and shoot yourself in the foot. I'll remove the references to SSL.